### PR TITLE
pino colada bug fix

### PR DESCRIPTION
**Pull request by davidmarkclements**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by davidmarkclements** [comment]
(also a template fix)

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "pino-colada": "^1.6.1",
```
Comment (mcollina) [inline package.json:57]: Can you bump pino-colada to v2.0.0?

**Comment by mcollina** [comment]
Also check out https://github.com/fastify/fastify-cli/pull/250.

cc @lundibundi

**Comment by lundibundi** [comment]
Did you verify this on an actual server output? I wrote seemingly the exact same code and while it did not crash it didn't prettify the output either. (Using `pino-colada@2`)

File: package.json (hunk 1 above)
Comment (davidmarkclements) [reply package.json]: done

**Comment by davidmarkclements** [comment]
> Did you verify this on an actual server output? I wrote seemingly the exact same code and while it did not crash it didn't prettify the output either. (Using `pino-colada@2`)

good spot, I think pino-coloda is broken:

image

**Comment by davidmarkclements** [comment]
here's the problem:

https://github.com/lrlna/pino-colada/blob/master/index.js#L26

we don't have `v` in latest pino

**Comment by davidmarkclements** [comment]
I've PR'd pino colada 

https://github.com/lrlna/pino-colada/pull/35

**Comment by davidmarkclements** [comment]
updated for latest pino-coloda,I think we're good to merge

**Pull request merged**